### DISCUSSION TITLE
Missing chunks in data - is it normal?

### DISCUSSION THREAD

**Discussion by fgdfgfthgr-fox** [janelia-cellmap/cellmap-segmentation-challenge#128] [Q&A] [answered]
I downloaded the data with the default `csc fetch-data` command. The total size of the data folder after downloading is 38.3GB, so matches the description. However, when I open the s2 level of jrc_cos7-1a in Fiji, this is what I see:
![2024-12-22 14-50-49屏幕截图](https://github.com/user-attachments/assets/1a05c68e-c300-4ec8-9eeb-f798b7a41f00)
Same goes for any other dataset I opened:
![2024-12-22 14-51-58屏幕截图](https://github.com/user-attachments/assets/c4174890-08f6-4828-b9eb-cd04d5499d76)
Are they supposed to be like that?

**Comment by mzouink** [comment, marked as answer]
Hi, yes, the default data fetcher, download only the raw blocks that are annotated and they have a GT blocks.

You can check the annotated blocks for each dataset [here](https://janelia-cellmap.github.io/cellmap-segmentation-challenge/available_data.html)
If you want to fetch more raw you can parametrize --raw-padding
e.g.
`csc fetch-data --raw-padding 128`

If the padding reach the total size of the raw the script will download the full raw dataseet. (check storage)
